src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -307,6 +311,83 @@ mesh::verify(const Node &n,
 }
 
 
+//-------------------------------------------------------------------------
+bool mesh::verify_multidomain(const Node &n,
+                              Node &info)
+{
+    info.reset();
+    bool res = true;
+
+    const std::string proto_name = "mesh";
+
+    if(!n.dtype().is_object() && !n.dtype().is_list())
+    {
+        log_error(info,proto_name,"not an object or a list");
+        res = false;
+    }
+    else
+    {
+        NodeConstIterator itr = n.children();
+        while(itr.has_next())
+        {
+            const Node &chld = itr.next();
+            std::string chld_name;
+            {
+                std::ostringstream oss;
+                if(n.dtype().is_object())
+                {
+                    oss << itr.name();
+                }
+                else
+                {
+                    oss << itr.index();
+                }
+                chld_name = oss.str();
+            }
+
+            if(!mesh::verify_unidomain(chld, info[chld_name]))
+            {
+                log_error(info,proto_name,
+                          "child " + chld_name + " is not a valid mesh");
+                res = false;
+            }
+        }
```

Comment:
This looks good, I think we should add a top level info entry that says we have a compliant multi domain  mesh.